### PR TITLE
Css grid classes

### PR DESCRIPTION
**Pull request by Smith-Steve**
image

### CODE REVIEW

File: css-grid-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column">
+        <h1>Choose your character!</h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column">
```
Comment (codingcodymiller) [inline css-grid-classes/index.html]: This `column` class doesn't seem to be doing anything. Make sure you use a column class with styling that matches the situation and a name that describes its purpose.

File: css-grid-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>CSS Grid Classes</title>
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column">
```
Comment (codingcodymiller) [inline css-grid-classes/index.html]: This `column` class doesn't seem to be doing anything. Make sure you use a column class with styling that matches the situation and a name that describes its purpose.

File: css-grid-classes/index.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      <div class="column-one-fourth">
+        <img src="images/ness.png" alt="Images">
+      </div><!-- Row 3 (4 columns) -->
+    </div>
+    <div class="row">
+      <div class="column-two-thirds">
+<h2>Donkey Kong</h2>
+<p>Donkey Kong is a series of video games featuring the advantures of a gorilla called Donkey Kong, conceived by Shigeru Miyamoto in 1980. The franchize consists mainly of two game genres, but also includes additional spin-off titles of various genres.</p>
+
+<p>The games of the first genre are mostly single-screen platform/action puzzle types, featuring Donkey Kong as the opponent in an industrial construction setting.</p>
```
Comment (codingcodymiller) [inline css-grid-classes/index.html]: Careful with the your indentation

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far!! Please see my notes

**Comment by Steve-A-Dore** [comment]
image

**Comment by Smith-Steve** [comment]
image

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**